src/lib/authui/main_dialog.rb

```diff
@@ -38,10 +38,11 @@ class MainDialog < UI::Dialog
 
         # Entry point can be :sssd, :ldapkrb, or :auto
         # In auto mode, there will be two change settings buttons.
-        def initialize(entry_point)
+        def initialize(entry_point, heading_caption_untranslated)
             super()
             textdomain 'auth-client'
             @entry_point = entry_point
+            @heading_caption_untranslated = heading_caption_untranslated
```

Comment:
I think this would work better:
```rb
def initialize(entry_point)
  super()
  textdomain 'auth-client'
  @entry_point = entry_point
  if entry_point == :ldapkrb,
    @heading_caption = _('LDAP and Kerberos Client')
  elsif entry_point == :sssd
    @heading_caption = _('User Logon Management')
  end
end
```